### ISSUE TITLE
No aside highlight on sub-pages

### ISSUE THREAD

**Issue by rdesorme** [Laravel-Backpack/theme-tabler#151]
# Bug report

### What I did

I created a CRUD for `/brands` and went to sub-pages (`/brands/create`, `/brands/1/show`, `/brands/1/edit`).

### What I expected to happen

The side menu should highlight the `/brands` menu item on all of these pages.

### What happened

It was highlighted only on the `/brands` URL, but not on `/brands/XXX`.

### What I've already tried to fix it

I replaced the code in [/resources/assets/js/tabler.js](https://github.com/Laravel-Backpack/theme-tabler/blob/main/resources/assets/js/tabler.js) :
```js
.filter(el => window.location.href.split("#")[0].split("?")[0] === el.href)
```
With :
```js
  .filter(el => {
    const href = window.location.href.split("#")[0].split("?")[0];
    return href === el.href || href.startsWith(el.href + '/');
  })
```

### Context to replicate the bug - Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

```
### PHP VERSION:
PHP 8.2.7 (cli) (built: Jun  8 2023 15:27:12) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.7, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.28.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.2.1
backpack/crud: 6.2.4
backpack/filemanager: 3.0.4
backpack/generators: v4.0.2
backpack/permissionmanager: 7.1.0
backpack/pro: 2.0.17
backpack/theme-tabler: 1.1.1

```

**Comment by karandatwani92** [comment]
Hey @rdesorme 

Can you explain with screenshots so I can understand better? Thanks

**Comment by rdesorme** [comment]
Hey,

Is this better ?

| What I got in Show / Create / Edit |What I expect in Show / Create / Edit
| --- | --- |
| ![image](https://github.com/Laravel-Backpack/theme-tabler/assets/31688687/4d385362-fde1-4111-9fb7-70ca8be52b0d) | ![image](https://github.com/Laravel-Backpack/theme-tabler/assets/31688687/5803cf8c-14ef-41ea-9d55-035d9cf3dcad) |

**Comment by pxpm** [comment]
Hey @rdesorme sorry for the delay on this issue. 

This should be fixed in 1.2.1 realease earlier today. 

Please re-open or comment if you are still experiencing issues
